packages/fuel-gauge/src/funding-transaction.test.ts

```diff
@@ -253,6 +253,7 @@ describe('Funding Transactions', () => {
     const getResourcesToSpend = vi.spyOn(sender, 'getResourcesToSpend');
 
     /**
+     * TODO: Update this comment with the new values
      * When estimating with only one UTXO the TX will require in total =~ 1887 of basetAssetId:
      * 1000 for the transfer and 887 for fees. However after funding it with 24 UTXOs the fee is
      * increased to =~ 3081. This happens because the bigger the TX becomes, more gas it will use.
```

Comment:
We can update the whole comment to:

```
/**
 * When estimating with only one UTXO, the transaction will require a total of 1523 of `baseAssetId`:
 * 1522 for the transfer and 1 for fees. However, after funding it with 254 UTXOs, the fee increases
 * to 3, bringing the total to 1525. This happens because as the transaction size grows, more gas is required.
 */
```